### PR TITLE
Exception tracking: no reported exceptions in the past week

### PR DESCRIPTION
**Pull request by davidwinter**
We want to ensure that applications are in a healthy state, and just installing and configuring exception tracking isn't enough. We want to ensure that teams regularly handle any uncaught exceptions to ensure the application is as reliable as possible in all known scenarios.

### CODE REVIEW

File: PRODUCTIONISATION.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 * [ ] 7.2 Alerts sent to relevant instant chat channel
 * [ ] 7.3 Alerts sent to relevant team email address
 * [ ] 7.4 UUID of request is sent as additional user data
+* [ ] 7.5 No reported unhandled exceptions in the past week
```
Comment (cybojenix) [inline PRODUCTIONISATION.md]: This is ambiguous.

Are we saying:
- No exceptions in Sentry that haven't been addressed within a week
- No exceptions in Sentry for at least a week

Comment (davidwinter) [reply PRODUCTIONISATION.md]: Updated with alternative wording, and added additional justification for the point.

File: PRODUCTIONISATION.md [hunk 2]
```diff
@@ -97,6 +97,8 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 * [ ] 7.2 Alerts sent to relevant instant chat channel
 * [ ] 7.3 Alerts sent to relevant team email address
 * [ ] 7.4 UUID of request is sent as additional user data
+* [ ] 7.5 No unresolved exceptions in the past week  
+  _Exceptions that have been thrown but not caught or handled safely could lead to larger problems for applications and teams. It's good for teams to have the discipline of regularly checking, or acting upon reported exceptions as they occur during development to prevent these from accruing. If code changes have been made to handle these, the exceptions should be marked as resolved in the tracking service. No reported exceptions in the past week is a positive sign of well managed exceptions._
```
Comment (cybojenix) [inline PRODUCTIONISATION.md]: The item is for "no unresolved exceptions", however the description goes on to talk about "no reported exceptions".

having "no exceptions" is an unreasonable thing to request.

File: PRODUCTIONISATION.md [hunk 3]
```diff
@@ -97,6 +97,8 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 * [ ] 7.2 Alerts sent to relevant instant chat channel
 * [ ] 7.3 Alerts sent to relevant team email address
 * [ ] 7.4 UUID of request is sent as additional user data
+* [ ] 7.5 No unresolved exceptions in the past week
```
Comment (cybojenix) [inline PRODUCTIONISATION.md]:
```suggestion
* [ ] 7.5 reported exceptions are resolved within a week
```
This orientates the item to be more around a process + discipline, which is more inline with what the description is talking about.

**Pull request merged**